views/index.erb

```diff
@@ -163,6 +163,101 @@
+        <p class="event__desc">A brief introduction to the day’s events.</p>
+      </li>
+      <li class="event">
+        <a class="event__link" href="/speakers#mike-north">
+          <time class="event__time" datetime="2015-06-16T09:30">09:30 am</time>
+          <h3 class="event__name">Compose all the things</h3>
+          <h4 class="event__speaker">Mike North</h4>
+        </a>
+      </li>
+      <li class="event">
+        <a class="event__link" href="/speakers#jay-phelps">
+          <time class="event__time" datetime="2015-06-16T10:10">10:10 am</time>
+          <h3 class="event__name">Graffiti: The Coveted Universal Web Component Format</h3>
+          <h4 class="event__speaker">Jay Phelps</h4>
+        </a>
+      </li>
+      <li class="event">
+        <a class="event__link" href="/speakers#adolfo-builes">
+          <time class="event__time" datetime="2015-06-16T10:50">10:50 am</time>
+          <h3 class="event__name">Writing code to update our code</h3>
+          <h4 class="event__speaker">Adolfo Builes</h4>
+        </a>
+      </li>
+      <li class="event">
+        <a class="event__link" href="/speakers#estelle-deblois">
+          <time class="event__time" datetime="2015-06-16T11:30">11:30 am</time>
+          <h3 class="event__name">Build Better Desktop Apps with Ember</h3>
+          <h4 class="event__speaker">Estelle DeBlois</h4>
+        </a>
+      </li>
+      <li class="event--general">
+        <time class="event__time" datetime="2015-06-16T12:10">12:10 pm</time>
+        <h3 class="event__name">Lunch</h3>
+      </li>
+      <li class="event">
+        <a class="event__link" href="/speakers#nathan-hammond">
+          <time class="event__time" datetime="2015-06-16T13:40">01:40 pm</time>
+          <h3 class="event__name">Building for Accessibility</h3>
+          <h4 class="event__speaker">Nathan Hammond</h4>
+        </a>
+      </li>
+      <li class="event">
+        <a class="event__link" href="/speakers#john-kleinschmidt">
+          <time class="event__time" datetime="2015-06-16T14:20">02:20 pm</time>
+          <h3 class="event__name">Taking Ember Offline</h3>
+          <h4 class="event__speaker">John Kleinschmidt</h4>
+        </a>
+      </li>
+      <li class="event--general">
+        <time class="event__time" datetime="2015-06-16T15:00">03:00 pm</time>
+        <h3 class="event__name">Afternoon Snack</h3>
+      </li>
+      <li class="event">
+        <a class="event__link" href="/speakers#stefan-penner">
+          <time class="event__time" datetime="2015-06-16T15:30">03:30 pm</time>
+          <h3 class="event__name">The observer tip-jar</h3>
+          <h4 class="event__speaker">Stefan Penner</h4>
+        </a>
+      </li>
+      <li class="event">
+        <a class="event__link" href="/speakers#igor-terzic">
+          <time class="event__time" datetime="2015-06-16T16:10">04:10 pm</time>
+          <h3 class="event__name">Ember Data 1.0-it’s here</h3>
```

Comment:
this should be how we do smart quotes
http://stackoverflow.com/questions/15776365/html-best-practices-should-i-use-rsquo-or-the-special-keyboard-shortcut
for reference we only need to escape & < and >
http://stackoverflow.com/questions/7381974/which-characters-need-to-be-escaped-on-html